lxd/instance/drivers/driver_qemu.go

```diff
@@ -2980,6 +2980,29 @@ func (d *qemu) addRootDriveConfig(mountInfo *storagePools.MountInfo, bootIndexes
 		TargetPath: rootDriveConf.TargetPath,
 	}
 
+	if d.storagePool.Driver().Info().Remote {
+		vol := d.storagePool.GetVolume(storageDrivers.VolumeTypeVM, storageDrivers.ContentTypeBlock, project.Instance(d.project, d.name), nil)
+
+		config := d.storagePool.ToAPI().Config
+
+		userName := config["ceph.user_name"]
+		if userName == "" {
+			userName = "admin"
```

Comment:
This value and the default value of `clusterName` would benefit from being defined as constants so they can be shared with `cephCreds()` (and any other places this default logic exists).